From 4977290ab4ab35258a6da9b13795c9b0f7894bf4 Mon Sep 17 00:00:00 2001
From: Diego Pino Garcia <dpino@igalia.com>
Date: Mon, 22 May 2023 19:58:50 -0700
Subject: [PATCH] [GLIB] Fix build error after 264196@main
 https://bugs.webkit.org/show_bug.cgi?id=256917

Reviewed by Michael Catanzaro.

Variable BWRAP_EXECUTABLE is only defined when BUBBLEWRAP_SANDBOX is
enabled.

* Source/WTF/wtf/glib/Sandbox.cpp:
(WTF::isInsideUnsupportedContainer):

Canonical link: https://commits.webkit.org/264395@main
Upstream-Status: Backport [https://github.com/WebKit/WebKit/commit/4977290ab4ab35258a6da9b13795c9b0f7894bf4]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 Source/WTF/wtf/glib/Sandbox.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/WTF/wtf/glib/Sandbox.cpp b/Source/WTF/wtf/glib/Sandbox.cpp
index 7d84e830ab33e..9b07bb8cb5a9b 100644
--- a/Source/WTF/wtf/glib/Sandbox.cpp
+++ b/Source/WTF/wtf/glib/Sandbox.cpp
@@ -36,6 +36,7 @@ bool isInsideFlatpak()
     return returnValue;
 }
 
+#if ENABLE(BUBBLEWRAP_SANDBOX)
 bool isInsideUnsupportedContainer()
 {
     static bool inContainer = g_file_test("/run/.containerenv", G_FILE_TEST_EXISTS);
@@ -64,6 +65,7 @@ bool isInsideUnsupportedContainer()
 
     return inContainer && !supportedContainer;
 }
+#endif
 
 bool isInsideSnap()
 {
